fe1-web/store/reducers/EventsReducer.ts

```diff
@@ -218,9 +218,9 @@ export const makeEventsMap = (laoId: string | undefined = undefined) => createSe
   },
 );
 
-export const makeEventGetter = (laoId: Hash | string, eventId: Hash | string) => {
+export const makeEventGetter = (laoId: Hash | string, eventId: Hash | string | undefined) => {
   const id = laoId.valueOf();
-  const evtId = eventId.valueOf();
+  const evtId = eventId?.valueOf();
```

Comment:
Why do we change the eventId to be optional?

Reply:
Because this function will be called using the last_tokenized_roll_call_id of an lao, which can be undefined. This way, the hook can be called without any conditions directly in the React component, so that the Rules of Hooks are not broken.

Reply:
I see, thanks for the explanation.